<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>SocketCAN</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>CAN device</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_SocketCAN_device">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>321</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>LAWICEL/SLCAN</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>57</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>COM port</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_COM_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>181</width>
         <height>24</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>57</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_speed">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>75</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>110</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>134</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>150</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>300</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>230400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>460800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>576000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>921600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1152000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3000000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4000000</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>57</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Databits</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>57</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop bits</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>57</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_Parity">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CAN bitrate</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_CAN_Bitrate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>10000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>125000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>800000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_Data_Bits">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_StopBits">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>90</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox_LoopBack">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Loopback mode</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBox_COM_port</tabstop>
  <tabstop>comboBox_speed</tabstop>
  <tabstop>comboBox_Parity</tabstop>
  <tabstop>lineEdit_SocketCAN_device</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
